tests/test-cases.js

```diff
@@ -1227,10 +1227,9 @@ var CUSTOM_TESTS = [
 				step: function(response, handle) {
 					updates.push(response.data[0]);
 					handle.abort();
-				},
-				complete: function() {
 					callback(updates);
```

Comment:
Ah, that makes sense -- that way we can actually check the results for correctness, not only that complete was called. If you could add that one other test, that would be stupendous!

@bluej100 Feel free to take a look at that when it comes in; also you may merge when you think it is ready and correct... thank you!